src/TestHelper/dir.props

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Do we expect that we will put other test stuff in here like PlatformDetection and our custom Asserts? We should actually consider doing that now that we have a test helper library here. 

As for a name I don't know if we need the name CoreFX included as I don't know that it buys us much, but I do wonder if we shouldn't have it under the Common folder so it doesn't look like a shipping library.